src/test/scala/dxWDL/base/ExtrasTest.scala

```diff
@@ -225,7 +229,8 @@ class ExtrasTest extends FlatSpec with Matchers {
 
     it should "throw IllegalArgumentException due to missing inputs in custom_reorg section" taggedAs(EdgeTest) in {
 
-        val app_id: String = "applet-123456"
+        // app_id is mummer nucmer app in project-FJ90qPj0jy8zYvVV9yz3F5gv
+        val app_id: String = "applet-FKfZpF002fp738k1Jqz8vX1G"
```

Comment:
let change all to camelcase 
app_id -> appId

Reply:
done